e2e/cmd/e2e.go

```diff
@@ -128,19 +86,16 @@ const (
 
 var allJobs = [...]job{jobKSM, jobKubelet, jobScheduler, jobEtcd, jobControllerManager, jobAPIServer}
 
-func execIntegration(pod v1.Pod, ksmPod *v1.Pod, dataChannel chan integrationData, wg *sync.WaitGroup, c *k8s.Client, logger *logrus.Logger) {
-	defer timer.Track(time.Now(), fmt.Sprintf("execIntegration func for pod %s", pod.Name), logger)
-	defer wg.Done()
```

Comment:
Here we got rid of a waitgroup and a channel. What were these doing?

Reply:
So, integration was executed in parallel on all listed pods. Commit message explains that IIRC.

But as integration runs fast and usually on one pod only, there is little value it seems in executing it asynchronously.